extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -174,6 +174,57 @@ export class Criteria {
   }
 }
 
+/**
+ * Regex for the width-selection portion of a srcset, so for the
+ * general grammar: (URL [NUM[w|x]],)*, this should express "NUMw".
+ * E.g. in "image1.png 100w, image2.png 50w", this matches "100w" and "50w"
+ */
+const srcsetWidthRe = /\s+([0-9]+)w(,|$)/g;
```

Comment:
See if we have this anywhere else? Looks like it should be a util

Reply:
There's the other `parseSrcset` that will give you a fully parsed object.

This use case for getting only the width-selection values is very particular, I'm not convinced this util is useful elsewhere.